src/main/java/org/opensearch/sdk/sample/helloworld/HelloWorldExtension.java

```diff
@@ -13,6 +13,7 @@
 import java.util.List;
 
 import org.opensearch.sdk.BaseExtension;
+import org.opensearch.sdk.EngineExtension;
```

Comment:
I see this import but don't see where it's used?

Reply:
Yes, I had used it to test locally and have removed it but import was left to remove. I have removed it now